algorithmiaio/langpacks
# ALGO-756 github actions CI template

Only useful when backed by github, automatically links to github actions.

TO TEST:
- Create a regular python37 algorithm on test, copy the `.github` directory and `TEST_CASES.json` into the new algorithm repo.
- Create or get an API Key that is able to call your algorithms + manage them
- Go to the github repo, click settings -> secrets and add a new secret called "api_key", paste the key from step 2 here
- if testing via test, edit .github/workflows/main.yml, replacing the variable for `api_address` with `https://api.test.algorithmia.com` instead of `https://api.algorithmia.com`
- commit your changes and check the actions tab for success/fail
